Tensile/TensileCreateLibrary.py

```diff
@@ -319,26 +315,11 @@ def buildSourceCodeObjectFile(CxxCompiler, outputPath, kernelFile):
     else:
       raise RuntimeError("Unknown compiler {}".format(CxxCompiler))
 
-    destCosList = []
-    if "PackageLibrary" in globalParameters and globalParameters["PackageLibrary"]:
-      for arch in archs:
-        ensurePath(os.path.join(destDir, arch))
-        archCoFilenames = [name for name in coFilenames if arch in name]
-        extractedCOs = [os.path.join(buildPath, name) for name in archCoFilenames]
-        destCOs = [os.path.join(destDir, arch, name) for name in archCoFilenames]
-        destCosList += destCOs
-        if globalParameters["PrintCodeCommands"]:
-          print1("# copy source code objects    : ", extractedCOs)
-          print1("# to dest source code objects : ", destCOs)
-        for (src, dst) in zip(extractedCOs, destCOs):
-          shutil.copyfile(src, dst)
-    else:
-      coFilenames = [name for name in coFilenames]
-      extractedCOs = [os.path.join(buildPath, name) for name in coFilenames]
-      destCOs = [os.path.join(destDir, name) for name in coFilenames]
-      destCosList += destCOs
-      for (src, dst) in zip(extractedCOs, destCOs):
-        shutil.copyfile(src, dst)
+    coFilenames = [name for name in coFilenames]
+    extractedCOs = [os.path.join(buildPath, name) for name in coFilenames]
+    destCosList = [os.path.join(destDir, name) for name in coFilenames]
+    for (src, dst) in zip(extractedCOs, destCosList):
+      shutil.copyfile(src, dst)
 
     return destCosList
```

Comment:
Inconsistent usage of code object acronym, `COs` vs `Cos`, not a huge deal, but I'm a fan of consistency.

Reply:
This is showing up as new code but it isn't. I changed indentation. Still happy to update naming.